thinc/compat.py

```diff
@@ -88,3 +88,13 @@
 
 
 has_gpu = has_cupy_gpu or has_torch_mps_gpu
+
+__all__ = [
+    "cupy",
+    "cupyx",
+    "torch",
+    "tensorflow",
+    "mxnet",
+    "h5py",
+    "os_signpost"
```

Comment:
Did you reformat with black? It should have caught this?

```suggestion
    "os_signpost",
```

Reply:
No - did so in https://github.com/explosion/thinc/pull/780/commits/312ded305fe53cb19fec24c27ffb6b3edadebd25. Thanks for the catch.